_rules/form-control-label-descriptive-cc0f0a.md

```diff
@@ -36,25 +45,19 @@ This rule applies to any HTML or SVG element that:
 
 **Note:** The `option` role is not part of the list of applicable roles, because it has a required context role that inherits from the `select` role. Furthermore, `option` does not meet the definition of a [User interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components). This means that [WCAG 2.1][wcag21] does not require it to have an [accessible name][].
 
-**Note**: [Labels][label] are not restricted to the `label` element of HTML and can be any element.
-
-**Note**: [Labels][label] have to be presented to all users. Therefore, [accessible names][accessible name] or elements that are not [included in the accessibility tree][] do not qualify as [labels][label] (unless coupled with similar content provided for other users).
+**Note**: [Labels][label] in WCAG are not restricted to the `label` element of HTML and can be any element. This rule is only concerned about actual `label` elements, and elements that are programmatically marked as [labels][label] via the `aria-labelledby` attribute.
 
 ## Expectation
 
-The [label][] describes the purpose of the test target. The [context][] of the [label][] can be used to differentiate the purpose from other test targets in the same test subject.
-
-**Note**: If the [label][] is not associated in a [programmatically determinable][] way with the form field element, this is likely a violation of [Success Criterion 1.3.1: Info and Relationships][sc131] but neither of this rule nor of [Success Criterion 2.4.6: Heading and Labels][sc246].
+Each test target, within its [visual context][], describes the purpose of the associated element.
```

Comment:
Instead of saying "within it's visual context", this should probably say "together with it's visual context".